examples/aws-provider/managed-resources/s3.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: s3.aws.crossplane.io/v1beta1
+kind: Bucket
+metadata:
+  name: test-channel-bucket
```

Comment:
can we have the name be consistent with what we have [here](https://github.com/awslabs/crossplane-on-eks/blob/d4d5a45b95fb841538ce9bd5a28fbc5f6dc5206d/examples/aws-provider/managed-resources/vpc.yaml#L9)

maybe `aws-provider-s3-bucket`?

Reply:
sounds good 👍